docs/diagrams/UML/TimeTableClassDiagram.puml

```diff
@@ -74,6 +78,8 @@ IncorrectCommand --|> "{abstract} \n Command"
 TimeTableAddCommand -|> "{abstract} \n TimeTableCommand"
 TimeTableDeleteCommand --|> "{abstract} \n TimeTableCommand"
 TimeTableViewCommand --|> "{abstract} \n TimeTableCommand"
+TimeTableResetCommand --|> "{abstract} \n TimeTableCommand"
```

Comment:
Try to replace line 79 with
`"{abstract} \n TimeTableCommand" <|-- TimeTableDeleteCommand`
and 81 with
`"{abstract} \n TimeTableCommand" <|-- TimeTableResetCommand`
to make the diagram look nicer